packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -3483,3 +3483,185 @@ void _debugDrawShadow(Canvas canvas, Path path, double elevation) {
     );
   }
 }
+
+/// The default shape of a Material 3 [Slider]'s value indicator.
+///
+/// See also:
+///
+///  * [Slider], which includes a value indicator defined by this shape.
+///  * [SliderTheme], which can be used to configure the slider value indicator
+///    of all sliders in a widget subtree.
+class DropSliderValueIndicatorShape extends SliderComponentShape {
```

Comment:
I was going to say that the theme file is an odd place for this, but given that all the other shapes are defined in this file it would be good to keep them together. I think the name is fine. I checked with the Material team and they didn't seem to have a canonical name that we could find, so `DropSliderValueIndicatorShape` it is 😎.

Reply:
Appreciate it!